BitwardenShared/UI/Auth/CompleteRegistration/CompleteRegistrationProcessor.swift

```diff
@@ -339,3 +354,13 @@ class CompleteRegistrationProcessor: StateProcessor<
         }
     }
 }
+
+// MARK: - CompleteRegistrationDelegate
+
+extension CompleteRegistrationProcessor: MasterPasswordUpdateDelegate {
+    func didUpdateMasterPassword(password: String) {
+        state.passwordText = password
+        state.retypePasswordText = password
+        coordinator.navigate(to: .dismissPresented)
```

Comment:
🎨 Curious if there any reason the navigation needs to happen here? More of a style thing, in other cases where we do this I think we usually have the delegate handle any updates it needs to do, but keep the navigation in the original processor.

Reply:
no specific reason, i guess i didn't think about it as much as i should have.  i can move it!